### DISCUSSION TITLE
Week9 - 於 phpMyAdmin 創建使用者帳號時跳出錯誤訊息 #1114 - The table 'db' is full

### DISCUSSION THREAD

**Discussion by york870198** [Lidemy/forum#135] [後端相關問題]
如題。
看起來使用者帳號是創建成功的，旁邊資料庫列表也確實出現同名的資料庫
![image](https://user-images.githubusercontent.com/9064855/120926611-4078e100-c710-11eb-9787-ebb120f8669e.png)
google 的結果是疑似跟預設上限空間有關？
想請問這具體是什麼狀況，以及如何解決。

**Comment by aszx87410** [comment]
查了一下確實看起來就是空間太小而導致，這是在你自己機器發生的嗎，還是在 mentor-program 那邊？

**Reply by york870198** [reply]
在我本機上，而且可以重現
我試過把使用者刪除掉然後重新創建，會發生一樣的事情
比較微妙的是我不確定那個「db」是指哪裡，因為我在介面上並沒有看到這個名稱的資料庫

**Reply by aszx87410** [reply]
照理來說如果出現這名字的話，應該是存在的可能性比較大，可以試試看直接下 query 然後 select * from db 看有沒有東西
